### DISCUSSION TITLE
Uwanted mpv behavior.

### DISCUSSION THREAD

**Discussion by jimford-sudo** [mpv-player/mpv#15321] [General]
I want to 'chew' through about a thousand video clips from a wildlife camera, to catch a certain event.
I've used 'find' with a regex, to narrow down  the likely files, and piped the output to mpv (- -print0 | xargs -0 mpv speed=2).

The problem is that the first 2 files show OK in a small window, but subsequent ones greatly exceed the size of the screen.

I'm using a 15 year old laptop. If it's a decoding problem, then why do the first  2 files show OK? - I'm way out of my depth here and could do with a bit of hand holding, please! 
Jim

**Comment by guidocella** [comment]
https://mpv.io/manual/master/#options-autofit-larger
https://mpv.io/manual/master/#options-auto-window-resize

**Comment by jimford-sudo** [comment]
Thanks guidocella  --auto-window-resize=no did the trick. (I thought it would be more complicated than that!)

To view the files faster I've tried --speed=2 (and other rates), but only get 15fps whatever I do.

``mpv --auto-window-resize=no --speed=2``

How can I increase the speed?
Jim

**Reply by guidocella** [reply]
`--hwdec --profile=fast`

**Reply by stonix015** [reply]
Generally, options which speed up mpv:

hwdec=auto -> Very important if you do not have activated it until yet. First, try this out 

If this does not help:
interpolation=no
scale=bilinear
dscale=bilinear
cscale=bilinear

The scaler are defaulted as lanczos .... and this feature consumes very much gpu-rendering power.

**Comment by jimford-sudo** [comment]
Many thanks for the helpful replies. Jim
